locales/ro.json

```diff
@@ -1127,5 +1135,93 @@
         "Prin utilizarea site-ului nostru declari că ai citit și ai înțeles această Politică de confidentțialitate a datelor cu caracter personal și îți exprimi acordul cu privire la felul în care colectăm și procesam datele cu caracter personal."
       ]
     }
+  },
+  "incometax": {
+    "title": "Redirecționare impozit",
+    "p1": {
+      "content": "Conform legii 227/2015 ( {0}) din Codul Fiscal, contribuabilii pot direcționa un procent de 3,5% din impozitul pe venit anual datorat pentru susținerea entitaților nonprofit, a unităților de cult, precum și pentru acordarea de burse private.",
+      "law_label": "link lege"
+    },
+    "p2": "În funcție de natura venitului, opțiunea se exercită prin completarea și depunerea la organul fiscal central competent a unui formular.",
+    "form": {
+      "title": "Formularul 230 „Cerere privind destinatia sumei reprezentand pana la 3,5% din impozitul anual datorat”",
```

Comment:
Some diacritics are missing in this title and in other places in the page like the next paragraph.

Reply:
Done